docs/client-connectors/databricks-client/install-databricks-connector.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 10
+title: Install Databricks Client Connector
+---
+
+This page explains how to install the Databricks Client Connector.
+
+## Prerequisites
+
+- Databricks Runtime 11.3 LTS(with Apache Spark 3.3.0)
+- Scala 2.12 or later
+- A Macrometa account with write permissions for streams. For more information about permissions, refer to [Permissions](../../account-management/permissions/).
+
+## Install the Macrometa Databricks Client Connector
+
+Do not start until all prerequisites are fulfilled.
+
+1. Obtain the JAR file for the connector using one of the following methods:
+   1. **Using the official GitHub package (Recommended for production)**: Download the pre-built JAR file from the official GitHub package for this repository. For example: app-0.0.1.jar
```

Comment:
@diana-macrometa - Please link the Github packages to the first item. Link: https://github.com/Macrometacorp/macrometa-connector-databricks/packages/1908628